### PR TITLE
chore(table): * global to th/td

### PR DESCRIPTION
**Pull request by mattnolting**
references #5728 
references #6369

### CODE REVIEW

File: src/patternfly/components/Table/table-grid.scss [hunk 1]
```diff
@@ -139,7 +139,7 @@
   border: none;
 
   // reset cell modifications
-  tr:where(.#{$table}__tr) > * {
+  tr:where(.#{$table}__tr) > :where(:is(th, td)) {
```
Comment (srambach) [inline src/patternfly/components/Table/table-grid.scss:142]: Can you just do this as `tr:where(.#{$table}__tr) > :where(th, td)` ?
Or even `:where(.#{$table}__tr) > :where(th, td)`?

File: src/patternfly/components/Table/examples/Table.md [hunk 2]
```diff
@@ -2801,8 +2801,8 @@ Width modifiers control the width of the columns. To control the responsive beha
 
 | Class | Applied to | Outcome |
 | -- | -- | -- |
-| `.pf-m-hidden{-on-[breakpoint]}` | `.pf-v6-c-table tr > *` | Hides a table cell at optional [breakpoint](/developer-resources/global-css-variables#breakpoint-variables-and-class-suffixes), or hides it at all breakpoints with `.pf-m-hidden`. **Note: Needs to apply to all cells in the column you want to hide.** |
-| `.pf-m-visible{-on-[breakpoint]}` | `.pf-v6-c-table tr > *` | Shows a table cell at optional [breakpoint](/developer-resources/global-css-variables#breakpoint-variables-and-class-suffixes). |
+| `.pf-m-hidden{-on-[breakpoint]}` | `.pf-v6-c-table tr > th/td` | Hides a table cell at optional [breakpoint](/developer-resources/global-css-variables#breakpoint-variables-and-class-suffixes), or hides it at all breakpoints with `.pf-m-hidden`. **Note: Needs to apply to all cells in the column you want to hide.** |
+| `.pf-m-visible{-on-[breakpoint]}` | `.pf-v6-c-table tr > th/td` | Shows a table cell at optional [breakpoint](/developer-resources/global-css-variables#breakpoint-variables-and-class-suffixes). |
```
Comment (srambach) [inline src/patternfly/components/Table/examples/Table.md:2805]: I'd probably write these as "`.pf-v6-c-table tr > th`,`.pf-v6-c-table tr > td`" - longer but no confusion about the slash

**Review by srambach — COMMENTED** [review]
All through - is there a reason to nest the :where(:is)) ? I think you could just use one or the other depending on whether you need the specificy, right?

File: src/patternfly/components/Table/table-grid.scss [hunk 3]
```diff
@@ -139,7 +139,7 @@
   border: none;
 
   // reset cell modifications
-  tr:where(.#{$table}__tr) > * {
+  tr:where(.#{$table}__tr) > :where(th, td) {
```
Comment (srambach) [inline src/patternfly/components/Table/table-grid.scss:142]: OK - and do we need specificity of 1 with `tr:where(.#{$table}__tr)` vs 0 for `:where(tr.#{$table}__tr)`?

**Review by srambach — APPROVED** [review]
🧨

**Review by mcoker — APPROVED** [review]
🚀

**Pull request merged**